### ISSUE TITLE
[Windows] User configuration dialog for image types, etc.

### ISSUE THREAD

**Issue by GerbilSoft** [GerbilSoft/rom-properties#52]
<a href="https://github.com/GerbilSoft"><img src="https://avatars2.githubusercontent.com/u/6764716?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [GerbilSoft](https://github.com/GerbilSoft)**
_Monday Feb 27, 2017 at 01:25 GMT_
_Originally opened as https://github.com/GerbilSoft/rom-properties.reworking/issues/52_

----

A configuration dialog needs to be added for each UI. This configuration dialog should allow the user to configure:
* Which image type is used for systems with multiple image types. Nintendo DS, for example, has icon, cover, and box.
* Enable downloads of high-resolution images if requested by the file browser. Disable this to save bandwidth. (It may be desired to be able to use high-resolution images if they're present, but fall back to "default" if they're not.)
* Disable image downloads entirely.
* For systems that support it, use the internal icon instead of other types for small sizes. (Need to define "small": 32x32, 48x48?)

This depends on #50 for image sizes.

**Comment by GerbilSoft** [comment]
<a href="https://github.com/DankRank"><img src="https://avatars2.githubusercontent.com/u/4050457?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [DankRank](https://github.com/DankRank)**
_Sunday Mar 05, 2017 at 00:26 GMT_

----

Maybe related: #21

**Comment by GerbilSoft** [comment]
<a href="https://github.com/GerbilSoft"><img src="https://avatars2.githubusercontent.com/u/6764716?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GerbilSoft](https://github.com/GerbilSoft)**
_Sunday May 14, 2017 at 01:39 GMT_

----

The Linux version will be postponed until the next release.
